docs/api-reference/carto/quadbin-heatmap-tile-layer.md

```diff
@@ -0,0 +1,104 @@
+
+## Usage 
+
+```tsx
+import DeckGL from '@deck.gl/react';
+import {QuadbinHeatmapTileLayer, quadbinTableSource} from '@deck.gl/carto';
+
+function App({viewState}) {
+  const data = quadbinTableSource({
+    accessToken: 'XXX',
+    connectionName: 'carto_dw',
+    tableName: 'carto-demo-data.demo_tables.quadbin'
+  });
+
+  const layer = new QuadbinHeatmapTileLayer({
+    data,
+    getWeight: d => d.properties.count
+  })
+
+  return <DeckGL viewState={viewState} layers={[layer]} />;
+}
+```
+
+## Installation
+
+To install the dependencies from NPM:
+
+```bash
+npm install deck.gl
+# or
+npm install @deck.gl/core @deck.gl/layers @deck.gl/carto
+```
+
+```js
+import {QuadbinHeatmapTileLayer} from '@deck.gl/carto';
+new QuadbinHeatmapTileLayer({});
+```
+
+To use pre-bundled scripts:
+
+```html
+<script src="https://unpkg.com/deck.gl@^9.0.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/carto@^9.0.0/dist.min.js"></script>
+
+<!-- or -->
+<script src="https://unpkg.com/@deck.gl/core@^9.0.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/layers@^9.0.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/geo-layers@^9.0.0/dist.min.js"></script>
+<script src="https://unpkg.com/@deck.gl/carto@^9.0.0/dist.min.js"></script>
+```
+
+```js
+new deck.carto.QuadbinHeatmapTileLayer({});
+```
+
+## Properties
+
+Inherits from all [Base Layer](../core/layer.md) and [CompositeLayer](../core/composite-layer.md) properties.
+
+#### `data` (TilejsonResult) {#data}
+
+Required. A valid `TilejsonResult` object.
+
+Use one of the following [Data Sources](./data-sources.md) to fetch this from the CARTO API:
+
+- [quadbinTableSource](./data-sources#quadbintablesource)
+- [quadbinQuerySource](./data-sources#quadbinquerysource)
+- [quadbinTilesetSource](./data-sources#quadbintilesetsource)
+
+### Render Options
+
+#### `radiusPixels` (number, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square") {#radiuspixels}
+
+* Default: `20`
+
+Radius of the heatmap blur in pixels, to which the weight of a cell is distributed.
+
+#### `colorDomain` (number[2], optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square") {#colordomain}
+
+* Default: `[0, 1]`
+
+Controls how weight values are mapped to the `colorRange`, as an array of two numbers [`minValue`, `maxValue`].
+
+When `colorDomain` is specified, a pixel with `minValue` is assigned the first color in `colorRange`, a pixel with `maxValue` is assigned the last color in `colorRange`, and any value in between is linearly interpolated. Pixels with weight less than `minValue` gradually fade out by reducing alpha, until 100% transparency representing `0`. Pixels with weight more than `maxValue` are capped to the last color in `colorRange`.
```

Comment:
> and any value in between is linearly interpolated.

Perhaps just "any value in between is interpolated"? I'm probably overthinking it, but I could see this implying interpolation in a linear color space, which the layer currently does not do.

> ... until 100% transparency representing `0`

Given signed colorDomain like `[-100, 100]`, I wouldn't be sure how to interpret this... do we need to say anything more about full transparency representing zero?

Reply:
The [description was wrong](https://github.com/visgl/deck.gl/blob/master/modules/carto/src/layers/heatmap.ts#L105C13-L105C23), I've [corrected](https://github.com/visgl/deck.gl/pull/8909/commits/3f92c14f1115239d8adc07ae5ffdcf8114dff454) it